### ISSUE TITLE
[UI - Wallet Stability]  Review / create `User Stories` for `Bridge` flows

### ISSUE THREAD

**Issue by noeliaSD** [status-im/status-desktop#15218]
Review / create the **User Stories** and **details** for `Bridge`. 
- US: https://www.notion.so/Launch-Bridge-Assets-flow-fa6f1408d5624a32bc0c9833263a54a4
- New design flow: https://www.figma.com/design/qiooiIE9bTgdBipKshS5uu/Bridge---Milestone-1?node-id=7401-84728&t=8NFjyjCnI8IVzLKR-1

Once they are ready, review which are the entry points already covered in the app and define needed tasks to work on the missing ones.

**Comment by noeliaSD** [comment]
It would make sense to work on it together with https://github.com/status-im/status-desktop/issues/15216

**Comment by noeliaSD** [comment]
@caybro please, sync with me before starting! I need to update the `User Stories` so, let me know before starting!

**Comment by noeliaSD** [comment]
@alexjba `Alisher` was the one building the new design and flows so maybe it makes sense to sync with him (also with `Ben`) when you create the  **User Stories** and start the _3 amigos_ round.

**Comment by alexjba** [comment]
The SendModal/Bridge will go through a re-design
